src/con_utils.ts

```diff
@@ -280,6 +280,9 @@ function parseConnectDsnAndArgs({
       } else {
         host = ["/run/edgedb", "/var/run/edgedb"];
       }
+      if (!admin) {
```

Comment:
@1st1 this is how respective code in python looks like. Was this omission intentional?